frontend/src/api/__tests__/imageStreams.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { k8sListResourceItems } from '@openshift/dynamic-plugin-sdk-utils';
+import { getNotebookImageStreams } from '~/api/k8s/imageStreams';
+import { ImageStreamModel } from '~/api/models';
+import { ImageStreamKind } from '~/k8sTypes';
+
+jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
+  k8sListResourceItems: jest.fn(),
+}));
+
+const k8sListResourceItemsMock = k8sListResourceItems as jest.Mock;
+
+describe('getNotebookImageStreams', () => {
+  it('should return notebook image streams', async () => {
+    const getNotebookImageStreamsMock = [
+      { metadata: { name: 'item 1' } },
+      { metadata: { name: 'item 2' } },
+    ] as ImageStreamKind[];
+
+    k8sListResourceItemsMock.mockResolvedValue(getNotebookImageStreamsMock);
+
+    const renderResult = await getNotebookImageStreams('namespace');
+
+    expect(k8sListResourceItemsMock).toHaveBeenCalledWith({
+      model: ImageStreamModel,
+      queryOptions: expect.objectContaining({
```

Comment:
Why use `objectContaining` here?

Doesn't it work as expected with a direct comparison:
```suggestion
      queryOptions: {
```

Reply:
it was a type, and Yes, It does work without `objectContaining`